### ISSUE TITLE
i keep getting this

### ISSUE THREAD

**Issue by ZwergTitan** [DistantDig/digs-dnd-origins#36]
22.09 03:14:38 [Server] main/ERROR There was a problem reading power file digs_dnd_origins:elf/high/ability_score_increase (skipping): Error reading data field at modifier.attribute: Identifier "spell_power:arcane" was not registered in registry "minecraft:attribute".
22.09 03:14:38 [Server] main/ERROR There was a problem reading power file digs_dnd_origins:elf/dark/ability_score_increase (skipping): Error reading data field at modifier.attribute: Identifier "spell_power:frost" was not registered in registry "minecraft:attribute".
22.09 03:14:38 [Server] main/ERROR There was a problem reading power file digs_dnd_origins:tiefling/ability_score_increase (skipping): Error reading data field at modifier.attribute: Identifier "spell_power:fire" was not registered in registry "minecraft:attribute".
22.09 03:14:38 [Server] main/ERROR Origin "digs_dnd_origins:elf/high" contained unregistered power: "digs_dnd_origins:elf/high/ability_score_increase"
22.09 03:14:38 [Server] main/ERROR Origin "digs_dnd_origins:elf/dark" contained unregistered power: "digs_dnd_origins:elf/dark/ability_score_increase"
22.09 03:14:38 [Server] main/ERROR Origin "digs_dnd_origins:tiefling" contained unregistered power: "digs_dnd_origins:tiefling/ability_score_increase"

**Comment by DistantDig** [comment]
Yeah, this is because there are a few abilities that won't load without certain mods. I haven't created a generator for the datapack yet, so everything is included even if the needed mods aren't present. This is a known issue, but it's not breaking your game or anything. I'll leave this open until I've officially fixed it
